<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lector de Gu√≠as</string> </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 75, 99);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(48, 92, 121);</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,10">
          <item>
           <widget class="QFrame" name="frame_bar">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="boton_cargar">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(224, 225, 221);</string>
               </property>
               <property name="text">
                <string>Cargar archivos</string>
               </property>
               </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="boton_deseleccionar">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(224, 225, 221);</string>
            </property>
            <property name="text">
             <string>Deseleccionar todo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="boton_seleccionar_todo">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(224, 225, 221);</string>
            </property>
            <property name="text">
             <string>Seleccionar Todo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="lista_imagenes">
            <property name="font">
             <font>
              <pointsize>14</pointsize> </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(224, 225, 221);</string>
            </property>
            <property name="uniformItemSizes">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(224, 225, 221);
border-radius: 10px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_nombre_archivo">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>16</pointsize> </font>
               </property>
               <property name="text">
                <string>Selecciona un archivo</string> </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum> </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_logo"> <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                 <size>
                  <width>100</width> <height>50</height>
                 </size>
                </property>
               <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>75</height>
                 </size>
                </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../assets/iconos.qrc">:/icons/logo_madre1.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(224, 225, 221);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="5,1">
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
                 <item>
                   <widget class="QFrame" name="frame_7">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QPushButton" name="boton_volver_imagen">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../assets/iconos.qrc">
                         <normaloff>:/icons/chevron_right.png</normaloff> </iconset>
                       </property>
                       <property name="toolTip">
                         <string>Imagen Anterior</string> </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                       <widget class="QPushButton" name="boton_siguiente_imagen">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                         <iconset resource="../assets/iconos.qrc">
                          <normaloff>:/icons/forward.png</normaloff> </iconset>
                       </property>
                       <property name="toolTip">
                         <string>Siguiente Imagen</string> </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 <item>
                   <widget class="QLabel" name="label_preview">
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Ignored"> <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>200</width> <height>200</height>
                      </size>
                     </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(200, 200, 200); border: 1px solid gray;</string> </property>
                    <property name="text">
                     <string>Previsualizaci√≥n</string> </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set> </property>
                    <property name="scaledContents">
                     <bool>false</bool> </property>
                   </widget>
                 </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(48, 92, 121);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="linea_edicion_texto">
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>14</pointsize> <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(224, 225, 221);</string>
                  </property>
                  <property name="placeholderText">
                    <string>Nuevo nombre (si falla el autom√°tico)</string> </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width> <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="boton_procesar">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(224, 225, 221);</string>
                  </property>
                  <property name="text">
                   <string>Procesar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="boton_guardar">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(224, 225, 221);</string>
                  </property>
                  <property name="text">
                   <string>Guardar Manual</string> </property>
                   <property name="toolTip">
                     <string>Guarda el nombre editado para el archivo seleccionado</string>
                   </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../assets/iconos.qrc"/>
 </resources>
 <connections/>
</ui>